test/support/test_helpers.ex

```diff
@@ -1,8 +1,10 @@
 defmodule ResearchResource.TestHelpers do
-  alias ResearchResource.Repo
+  alias ResearchResource.{Repo, User}
+  alias Plug.Conn
```

Comment:
To be honest @SimonLab, I'm not too sure, but I think it will be fine.

[`Plug.Conn.assign`](https://github.com/elixir-lang/plug/blob/v1.3.4/lib/plug/conn.ex#L286) seems like a fairly simple pure function that shouldn't have any risks with it

@SimonLab Let me know what your preference would be for dealing with this issue, in my previous commit I passed down `assign` to be used in the `login_user` function, would you prefer something like that?